test/e2e-api/content/newsletters.test.js

```diff
@@ -0,0 +1,43 @@
+const {agentProvider, fixtureManager, matchers} = require('../../utils/e2e-framework');
+
+const newsletterSnapshot = {
+    id: matchers.anyObjectId,
+    uuid: matchers.anyUuid,
+    name: matchers.anyString,
```

Comment:
You only need to add matchers for data that changes. So id, uuid, and dates.

Reply:
Updated, thanks.